doc/rtd/reference/base_config_reference.rst

```diff
@@ -320,15 +320,17 @@ On an Ubuntu system, :file:`/etc/cloud/cloud.cfg` should look similar to:
 
     # The modules that run in the 'config' stage
     cloud_config_modules:
+    - wireguard
```

Comment:
Did something weird with a rebase happen, or did you just realize that we didn't have everything here?